src/olympia/migrations/907-drop-password-and-confirmationcode.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users
+    DROP COLUMN `password`,
+    DROP COLUMN `confirmationcode`;
```

Comment:
I think this is likely to produce some kind of load or hanging on prod, let's see how it goes on dev and stage and keep @jasonthomas in the loop that there's a bigger migration coming :)